Front-end/store/src/app/core/services/basket.service.ts

```diff
@@ -27,7 +27,7 @@ export class BasketService {
 
   public getBasket(): void {
     const headers = {
```

Comment:
Authorization headers shouldn't generate in BasketService methods. https://indepth.dev/tutorials/angular/authentication-token-interceptor